package.json

```diff
@@ -105,13 +105,17 @@
 		"rules": {
 			"no-alert": "off",
 			"import/no-unassigned-import": "off",
-			"import/no-unresolved": "warn"
+			"import/no-unresolved": "warn",
+			"import/named": "warn"
```

Comment:
https://github.com/benmosher/eslint-plugin-import/issues/1341

Reply:
That rule has other problems too. Disabled: https://github.com/xojs/xo/commit/ee145cbe719073aeccd349af4bb85ddc542a2818